Create or Update alias API supports must_exist parameter (#6245)
opensearch-project/documentation-website: _im-plugin/index-alias.md

GET /_alias/alias1
GET /index-1/_alias/alias1
```
{% include copy-curl.html %}

> To get the indexes' mappings and settings information that the alias references, run the following command:

Reviewer: ```suggestion
To get the indexes' mappings and settings information referenced by the alias, run the following command:
```

```json
GET alias1
```
{% include copy-curl.html %}

## Add or remove indexes

> You can perform multiple actions in the same `_aliases` operation. For example, the following command removes `index-1` and adds `index-2` to `alias1`:

Reviewer: ```suggestion
You can perform multiple actions using the same `_aliases` operation. For example, the following command removes `index-1` and adds `index-2` to `alias1`:
```

  ]
}
```
{% include copy-curl.html %}

The `remove` action also supports the `must_exist` parameter. If the parameter is set to `true` and the specified alias does not exist, an exception is thrown. If the parameter is set to `false`, then no action is taken if the specified alias does not exist. The default value for `must_exist` is `null`. An exception will be thrown only if none of the specified aliases exist. 

> The following POST request uses the `remove` action, with `must_exist` parameter set to `true`:

Reviewer: ```suggestion
The following POST request uses the `remove` action with the `must_exist` parameter set to `true`:
```

}
```

> In addition, `remove` action supports `must_exist` paramater, if it is `true` and the specified alias doesn't exist, an exception will be thrown, if it is `false`, nothing happens if the specified alias doesn't exist. The default value of `must_exist` is `null`, only if all of the specified aliases don't exist, an exception will be thrown.

Reviewer 2: ```suggestion
The `remove` action also supports the `must_exist` parameter. If the parameter is set to `true` and the specified alias does not exist, an exception is thrown. If the parameter is `false`, then no action is taken if the specified alias does not exist. The default value for `must_exist` is `null`. An exception will be thrown only if none of the specified aliases exist.
```

HEAD /_alias/alias1/
HEAD index-1/_alias/alias1/
```
{% include copy-curl.html %}

## Add aliases at index creation

> You can add an index to an alias as you create the index like that shown in the following PUT request:

Reviewer: ```suggestion
You can add an index to an alias as you create the index, as shown in the following PUT request:
```

  }
}
```
{% include copy-curl.html %}

## Create filtered aliases

> You can create a filtered alias to access a subset of documents or fields from the underlying indexes. This command adds only a specific timestamp field to `alias1`. The following shows a POST request example:

Reviewer: ```suggestion
You can create a filtered alias to access a subset of documents or fields in the underlying indexes. This command adds only a specific timestamp field to `alias1`. The following shows an example POST request:
```

DELETE index-1/_alias/alias1
```

> After you run the request above, `alias1` no longer refers to `index-1`, but still refers to `index-2`.

Reviewer 2: ```suggestion
After running the previous request, `alias1` no longer refers to `index-1`, but still refers to `index-2`.
```
